alcs-api/src/application/application.service.ts

```diff
@@ -39,7 +39,9 @@ export class ApplicationService {
     applicationEntity.title = application.title;
     applicationEntity.body = application.body;
     applicationEntity.statusUuid = application.statusUuid;
-    applicationEntity.assigneeUuid = application.assigneeUuid;
+    applicationEntity.assigneeUuid = application.assigneeUuid
+      ? application.assigneeUuid
+      : null;
```

Comment:
Why did this need to be changed?

Reply:
you need to set value to null explicitly otherwise if it is undefined it is ignored